apps/web/src/design-system/table/Table.tsx

```diff
@@ -79,7 +100,7 @@ export function Table({
         : {}),
     } as any,
     usePagination
-  );
+  ) as UseTableProps;
```

Comment:
adjusted types for the `useTable` hook... casted it here to the `UseTableProps` because there is no another way to do this when the hook uses other plugins, like `usePagination`